pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -409,6 +409,11 @@ func (gce *GCECloud) ScrubDNS(nameservers, searches []string) (nsOut, srchOut []
 	return nameservers, srchOut
 }
 
+// HasClusterID returns true if the cluster has a clusterID
+func (gce *GCECloud) HasClusterID() bool {
```

Comment:
I see. It's [g](https://github.com/kubernetes/kubernetes/blob/088141ca3a193feab943375bcc564db1a66ef58a/pkg/cloudprovider/providers/gce/gce.go#L159-L172) at one part of the file and [gce](https://github.com/kubernetes/kubernetes/blob/088141ca3a193feab943375bcc564db1a66ef58a/pkg/cloudprovider/providers/gce/gce.go#L342-L396) for the rest. Your code is fine here, feel free to ignore the lint message :)